src/Sylius/Behat/spec/Context/Ui/Admin/ManagingCountriesContextSpec.php

```diff
@@ -212,4 +215,22 @@ function it_throws_not_equal_exception_if_country_code_field_is_not_disabled(Upd
         $countryUpdatePage->isCodeFieldDisabled()->willReturn(false);
         $this->shouldThrow(\InvalidArgumentException::class)->during('theCodeFieldShouldBeDisabled');
     }
+
+    function it_asserts_that_a_country_has_the_province(CountryInterface $country, UpdatePageInterface $countryUpdatePage)
+    {
+        $country->getId()->willReturn(1);
+        $countryUpdatePage->isOpen(['id' => 1])->willReturn(true);
+        $countryUpdatePage->isThereProvince('Scotland')->willReturn(true);
+
+        $this->countryShouldHaveProvince($country, 'Scotland');
```

Comment:
@pamil sadly, I don't know. I guess support for language in Euro Tr**u**ck Simulator 2 is more important than support for language... in the world. :earth_americas:

Nevertheless, I suppose to avoid such a fascinating discussion, we could use different example than controversial ones :smile: